BackofficeBundle/Resources/public/js/backboneRouter.js

```diff
@@ -166,20 +166,17 @@ var OrchestraBORouter = Backbone.Router.extend({
 
   generateUrl: function(routeName, paramsObject)
   {
+    var optionalParam = /\(([^)]*):([^)]*)\)/g;
```

Comment:
the pattern is an arbitrary choice. Could be language.fr or languagefr or anything else. But in order to have no conflict, I add '_'. Here we are in remplacement process, so separator character has no importance. We could choose to change separator in route, if we got conflict (i.e. language_fr is an entity) without change resolve routing process. @nicolasThal